ci/build-push-multiplatform.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+if ! docker buildx ls | grep arm
+then
+    echo "Your Buildx seems to lack ARM architecture support"
+    exit 1
+fi
+
+docker buildx build \
+    --push \
+    --platform=linux/amd64 \
+    --build-arg FLUENTD_ARCH="" \
```

Comment:
I guess this assumes that the build is being triggered on `amd64`, can't we explicitly say that we build for amd64 here?

**EDIT** Oh i guess the fluentd images naming convention makes it so :/ 🤔

Reply:
Yeah the Fluentd image tags are a bit sucky in that the amd64 version has no architecture-specific suffix, so for amd64 we need to pass empty string as the value for the build arg.